src/utils/fetchData.js

```diff
@@ -1,19 +1,21 @@
 import queryPerfData from '@mozilla-frontend-infra/perf-goggles';
-import { BENCHMARKS, CONFIG } from '../config';
+import linux64 from '../configuration/js-team/linux64';
```

Comment:
I see what you're doing here. Remove this line completely.
I will add a comment below.

Reply:
Wouldn't this work?
```suggestion
import config from '../configuration/js-team';
```
See next comment.